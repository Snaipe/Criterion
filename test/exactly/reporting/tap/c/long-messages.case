[act]

% long-messages.c.bin --tap

[assert]

stdout
  -transformed-by replace_version
  equals -contents-of expected/long-messages.txt
