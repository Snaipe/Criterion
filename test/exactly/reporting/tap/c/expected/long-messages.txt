TAP version 13
1..1
# Criterion <VERSION>
#
# Running 1 tests from sample
not ok - sample::long_msg 
  ---
  assertions: 1
  failures:
  - long-messages.c:4: |+
      Assertion failed: This is
      A long message
      Spawning multiple lines.
      Formatting is respected.
  ...
